src/main/java/frc/team2412/robot/commands/intake/IntakeBallCommand.java

```diff
@@ -0,0 +1,20 @@
+package frc.team2412.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import frc.team2412.robot.commands.index.IndexCommand;
+import frc.team2412.robot.subsystem.IndexSubsystem;
+import frc.team2412.robot.subsystem.IntakeSubsystem;
+
+public class IntakeBallCommand extends ParallelCommandGroup {
+    private final IndexSubsystem indexSubsystem;
+    private final IntakeSubsystem intakeSubsystem;
+
+    public IntakeBallCommand(IndexSubsystem indexSubsystem, IntakeSubsystem intakeSubsystem) {
+        this.indexSubsystem = indexSubsystem;
+        this.intakeSubsystem = intakeSubsystem;
+        addCommands(
```

Comment:
Index will be used when shooting and when intaking, so adding requirements is very important to make sure there no collisions.